### DISCUSSION TITLE
Global Search

### DISCUSSION THREAD

**Discussion by patzobil** [docmost/docmost#165] [Ideas]
Add a global search function on top of the Space Search for easier finding of notes.
Also add a Keyboard Shortcut to the Global Search and the Spaces Search functions (Something like Ctrl + E for space search and Ctrl + Shift + E for global? )

**Comment by patzobil** [comment]
Anyone any ideas to support this? I know it could be tricky only to search in those spaces where the usert has access to...

**Reply by Philipinho** [reply]
It shouldn't be difficult to implement. I will eventually get to it.

**Reply by Philipinho** [reply]
This is available in the latest version.
